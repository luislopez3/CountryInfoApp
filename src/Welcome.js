import React from "react";

export default function Welcome() {
    return (
      <div>
        <h1>Welcome!</h1>
        <p>
          This app uses nested routes to display data for all the Countries around
          the World.
        </p>
        <p>To begin, click the "Regions" link.</p>
      </div>
    );
  }