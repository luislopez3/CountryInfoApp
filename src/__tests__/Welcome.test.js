import React from "react";
import { create } from "react-test-renderer";
import { MemoryRouter } from "react-router";
import Welcome from "../Welcome.js";

describe("Show Details component", () => {
  test("Matches the snapshot", () => {
    const wrapper = create(
      <MemoryRouter>
        <Welcome />
      </MemoryRouter>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
