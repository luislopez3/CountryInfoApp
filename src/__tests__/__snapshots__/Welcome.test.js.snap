// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Details component Matches the snapshot 1`] = `
<div>
  <h1>
    Welcome!
  </h1>
  <p>
    This app uses nested routes to display data for all the Countries around the World.
  </p>
  <p>
    To begin, click the "Regions" link.
  </p>
  <p>
    Or you can browse through "All Countries".
  </p>
</div>
`;
